src/main/java/com/everis/cars/boundary/CarResource.java

```diff
@@ -53,6 +69,15 @@ public Response createCar(
 		final Car car
 	);
 
+	/**
+	 * Update a car by ID
+	 * 
+	 * @param carId the car ID to be updated
+	 * @param car the car object to update the registry
+	 * @throws CarNotFoundException
```

Comment:
is it really throwing the exception? worth maybe mentioning error codes here

Reply:
even mentioning how BeanValConstrainViolationExceptionMapper will intercept returning a bad response?

Reply:
> is it really throwing the exception? worth maybe mentioning error codes here

explained better where and why this exception is throwing.



> even mentioning how BeanValConstrainViolationExceptionMapper will intercept returning a bad request?

explained in comments how I catch the bean validations and the return sended